design-documents/graph-ql/coverage/returns.graphqls

```diff
@@ -0,0 +1,197 @@
+type Query{
+    # See https://github.com/magento/architecture/blob/master/design-documents/graph-ql/framework/attributes-metadata.md
+    pageSpecificCustomAttributes(
```

Comment:
Is this approach already established? I'm not a big fan of having "page" concept in gql

Reply:
We agreed on this some time ago (see the link on line 2), but this is the first implementation. So we can easily reconsider it if you have better ideas.